src/config.rs

```diff
@@ -62,6 +62,7 @@ pub enum Config {
     SysVersion,
     #[strum(serialize = "sys.msgsize_max_recommended")]
     SysMsgsizeMaxRecommended,
+    ChatMode,
```

Comment:
`ChatMode` aka `chat_mode` sounds a bit too generic.
i would suggest sth. as `messageid_prefix` or even `rfc724_mid_prefix`.
the setting should be marked as experimental and will go away as soon a `chat$` becomes the default in half a year or so.